src/config.h

```diff
@@ -130,3 +130,5 @@ extern CONFIG g_Config;
 
 bool Config_ReadFromJSON(const char *json);
 bool Config_Read(void);
+void Config_Init(void);
+bool Config_Write(void);
```

Comment:
Missing end of the line character

Reply:
Resolved in [4ee0f5a](https://github.com/rr-/Tomb1Main/pull/637/commits/4ee0f5a3487067af292d756b533979a5b9ffd017)